app/views/quotes/index.html.erb

```diff
@@ -9,7 +9,11 @@
       </footer>
     </blockquote>
     <div class="created">
-      <%= time_ago_in_words(quote.created_at) %> ago
+    <% if quote.created_at %>
```

Comment:
Kevin says: "hey, what did you mean by that? I know these quotes have `nil` on create_at -- but I'm not sure how fix this in the database., Can you point me in the right direction? "